From 5345194283ec9335701d8a89d40aca4643d5c8ac Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 19 Sep 2023 14:19:52 +0800
Subject: [PATCH 2/2] fix build failure on gcc 13

./tensorflow/tsl/lib/io/cache.h:20:1: note: 'uint64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   19 | #include "tensorflow/tsl/platform/stringpiece.h"
  +++ |+#include <cstdint>

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/tsl/lib/io/cache.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tensorflow/tsl/lib/io/cache.cc b/tensorflow/tsl/lib/io/cache.cc
index ee5a3712ba8..00db8b99f77 100644
--- a/tensorflow/tsl/lib/io/cache.cc
+++ b/tensorflow/tsl/lib/io/cache.cc
@@ -13,6 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+#include <stdint.h>
 #include "tensorflow/tsl/lib/io/cache.h"
 
 #include <assert.h>
-- 
2.27.0

